frontend/public/components/utils/tile-view-page.jsx

```diff
@@ -399,13 +429,13 @@ const setURLParams = (params) => {
   history.replace(`${url.pathname}${searchParams}`);
 };
 
-export const updateURLParams = (filterName, value) => {
+export const updateURLParams = (paramName, value) => {
```

Comment:
Add types

Reply:
It's a jsx file

Reply:
Oh missed that. Ignore this comment then :P